### PR TITLE
Latest version change to 18

### PR DESCRIPTION
**Pull request by karianna**
# Checklist

- [ ] You added tests to cover the change
- [ ] `mvn clean install` build and test completes
- [ ] You changed or added to the documentation

### CODE REVIEW

**Review by jerboaa — APPROVED** [review]

**Comment by karianna** [comment]
Changing to draft until I can test this locally.

**Comment by karianna** [comment]
Git rebase screwed me again, closing this.

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -11,14 +11,18 @@ Since the GitHub API is rate limited we use MongoDB as a caching mechanism.
 
 There are two main branches in the project:
 
+<<<<<<< HEAD
+- `main`
+=======
 - `master`
+>>>>>>> 1abe08e087738755e3fdd9194de4e6d6f4f1a880
```
Comment (jerboaa) [inline CONTRIBUTING.md:18]: @karianna Left-over merge conflict hunks?

Comment (jerboaa) [reply CONTRIBUTING.md:18]: ah, nvm.

**Pull request closed without merging**